### ISSUE TITLE
username.sync() returns ubuntu?

### ISSUE THREAD

**Issue by skistaddy** [sindresorhus/username#6]
When I set a variable to `username.sync()`, it is returning `ubuntu` instead of my username. Fix?

**Comment by sindresorhus** [comment]
What's the output of running:

```
$ echo 1:$SUDO_USER 2:$LOGNAME 3:$USER 4:$LNAME 5:$USERNAME 6:$(id -un)
```

**Comment by skistaddy** [comment]
@sindresorhus `1: 2:ubuntu 3:ubuntu 4: 5: 6:ubuntu`

**Comment by skistaddy** [comment]
I'm using https://c9.io signed in with my GitHub account.

**Comment by sindresorhus** [comment]
Hmm, not much I can do when those are wrong.

Can you share the output of `$ node -p 'process.env'`? (Scrub any sensitive info)

**Comment by skistaddy** [comment]
{ APACHE_PID_FILE: '/home/ubuntu/lib/apache2/run/apache2.pid',
  MANPATH: '/home/ubuntu/.nvm/versions/node/v4.4.3/share/man:/usr/local/rvm/rubies/ruby-2.3.0/share/man:/usr/local/man:/usr/local/share/man:/usr/share/man:/usr/local/rvm/man',
  rvm_bin_path: '/usr/local/rvm/bin',
  C9_SHARED: '/mnt/shared',
  C9_FULLNAME: 'Baran Skistad',
  GEM_HOME: '/usr/local/rvm/gems/ruby-2.3.0',
  NVM_CD_FLAGS: '',
  APACHE_RUN_USER: 'ubuntu',
  SHELL: '/bin/bash',
  TERM: 'screen',
  IRBRC: '/usr/local/rvm/rubies/ruby-2.3.0/.irbrc',
  SSH_CLIENT: '::ffff:10.240.0.221 58712 22',
  ISOUTPUTPANE: '0',
  NVM_PATH: '/home/ubuntu/.nvm/versions/node/v4.4.3/lib/node',
  C9_PORT: '8080',
  METEOR_IP: '0.0.0.0',
  MY_RUBY_HOME: '/usr/local/rvm/rubies/ruby-2.3.0',
  PHPRC: '/home/ubuntu/workspace',
  LC_ALL: 'C.UTF-8',
  NVM_DIR: '/home/ubuntu/.nvm',
  USER: 'ubuntu',
  LD_LIBRARY_PATH: '~/.c9/local/lib',
  _system_type: 'Linux',
  rvm_path: '/usr/local/rvm',
  C9_UID: '951011',
  TMUX: '/tmp/tmux-1000/cloud91.8,2072,0',
  C9_IP: '0.0.0.0',
  rvm_prefix: '/usr/local',
  APACHE_LOG_DIR: '/home/ubuntu/lib/apache2/log',
  PATH: '/home/ubuntu/.nvm/versions/node/v4.4.3/bin:/usr/local/rvm/gems/ruby-2.3.0/bin:/usr/local/rvm/gems/ruby-2.3.0@global/bin:/usr/local/rvm/rubies/ruby-2.3.0/bin:/mnt/shared/bin:/home/ubuntu/workspace/node_modules/.bin:/home/ubuntu/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/shared/sbin:/opt/gitl:/opt/go/bin:/mnt/shared/c9/app.nw/bin:/usr/local/rvm/bin',
  C9_USER: 'bjskistad',
  HGUSER: 'Baran Skistad',
  NVM_NODEJS_ORG_MIRROR: 'https://nodejs.org/dist',
  PWD: '/home/ubuntu/workspace',
  APACHE_RUN_GROUP: 'ubuntu',
  LANG: 'C',
  NODE_PATH: '/mnt/shared/lib/node_modules',
  _system_arch: 'x86_64',
  TMUX_PANE: '%0',
  _system_version: '14.04',
  C9_SH_EXECUTED: '1',
  rvm_version: '1.27.0 (latest)',
  HOME: '/home/ubuntu',
  SHLVL: '4',
  C9_PID: '3199451',
  GOROOT: '/opt/go',
  LANGUAGE: 'C.UTF-8',
  C9_PROJECT: 'jel',
  LOGNAME: 'ubuntu',
  C9_EMAIL: 'bskist01@isd77.k12.mn.us',
  GEM_PATH: '/usr/local/rvm/gems/ruby-2.3.0:/usr/local/rvm/gems/ruby-2.3.0@global',
  SSH_CONNECTION: '::ffff:10.240.0.221 58712 ::ffff:172.17.35.22 22',
  LESSOPEN: '| /usr/bin/lesspipe %s',
  GOPATH: '/home/ubuntu/workspace',
  NVM_BIN: '/home/ubuntu/.nvm/versions/node/v4.4.3/bin',
  EMAIL: 'bskist01@isd77.k12.mn.us',
  NVM_IOJS_ORG_MIRROR: 'https://iojs.org/dist',
  PORT: '8080',
  METEOR_PORT: '8080',
  IP: '0.0.0.0',
  APACHE_LOCK_DIR: '/home/ubuntu/lib/apache2/lock',
  APACHE_RUN_DIR: '/home/ubuntu/lib/apache2/run',
  C9_HOSTNAME: 'jel-bjskistad-1.c9users.io',
  LESSCLOSE: '/usr/bin/lesspipe %s %s',
  RUBY_VERSION: 'ruby-2.3.0',
  rvm_silence_path_mismatch_check_flag: '1',
  _system_name: 'Ubuntu',
  'BASH_FUNC__gnomeopen%%': '() {  if [ -e "$@" ]; then\n c9 "$@";\n return;\n fi;\n command xdg-open "$@"\n}',
  'BASH_FUNC__xdgopen%%': '() {  if [ -e "$@" ]; then\n c9 "$@";\n return;\n fi;\n command xdg-open "$@"\n}',
  _: '/home/ubuntu/.nvm/versions/node/v4.4.3/bin/node' }

**Comment by sindresorhus** [comment]
Fixed in 2.2.0 :)

**Comment by skistaddy** [comment]
Thanks! (Nice job btw.)
